lib/modules/datasource/metadata-manual.ts

```diff
@@ -1,6 +1,28 @@
 // Use this object to define changelog URLs for packages
 // Only necessary when the changelog data cannot be found in the package's source repository
 export const manualChangelogUrls: Record<string, Record<string, string>> = {
+  maven: {
+    'org.spockframework:spock-core':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+    'org.spockframework:spock-bom':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+    'org.spockframework:spock-groovy2-compat':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+    'org.spockframework:spock-guice':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+    'org.spockframework:spock-junit4':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+    'org.spockframework:spock-specs':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+    'org.spockframework:spock-spring':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+    'org.spockframework:spock-tapestry':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+    'org.spockframework:spock-testkit':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+    'org.spockframework:spock-unitils':
+      'https://github.com/spockframework/spock/blob/master/docs/release_notes.adoc',
+  },
```

Comment:
Maybe there could be a mechanism to do it with the `org.spockframework` (or `org.spockframework:spock-`) prefix?

Reply:
Btw, the better URL would be:
> https://spockframework.org/spock/docs/$VERSION_NUBER/release_notes.html

But I don't know if it could be parameterized? In addition, it is even slightly more complicated, as a version might be `2.3-groovy-3.0` or `2.3-groovy-4.0` for Spock 2.3 :-/.

I will see, if we can add a link "current" or "latest" redirecting to the latest version.